data/factory-sets.json

```diff
@@ -6686,226 +6496,283 @@
+				"ability": ["Overgrow"],
+				"evs": {"hp": 0,"atk": 252,"def": 0,"spa": 0,"spd": 4,"spe": 252},
+				"nature": "Jolly",
+				"moves": [["Swords Dance"], ["Spirit Shackle"], ["Leaf Blade"], ["Substitute", "Sucker Punch"]]
+			}]
+		},
+		"dhelmise": {
+			"flags": {},
+			"sets": [{
+				"species": "Dhelmise",
 				"gender": "",
 				"item": ["Leftovers"],
-				"ability": ["Synchronize"],
-				"evs": {"hp": 252, "atk": 0, "def": 0, "spa": 0, "spd": 212, "spe": 44},
-				"nature": "Calm",
-				"moves": [["Moonlight"], ["Toxic"], ["Taunt"], ["Foul Play"]]
+				"ability": ["Steelworker"],
+				"evs": {"hp": 172,"atk": 252,"def": 0,"spa": 0,"spd": 0,"spe": 84},
+				"nature": "Adamant",
+				"moves": [["Rest"], ["Shadow Claw"], ["Anchor Shot"], ["Rapid Spin"]]
 			}]
 		},
-		"chesnaught": {
+		"donphan": {
 			"flags": {},
 			"sets": [{
-				"species": "Chesnaught",
+				"species": "Donphan",
 				"gender": "",
-				"item": ["Salac Berry"],
-				"ability": ["Overgrow"],
-				"evs": {"hp": 12, "atk": 244, "def": 0, "spa": 0, "spd": 0, "spe": 252},
-				"nature": "Jolly",
-				"moves": [["Substitute"], ["Belly Drum"], ["Seed Bomb"], ["Earthquake"]]
-			}, {
-				"species": "Chesnaught",
+				"item": ["Leftovers"],
+				"ability": ["Sturdy"],
+				"evs": {"hp": 252,"atk": 252,"def": 4,"spa": 0,"spd": 0,"spe": 0},
+				"nature": "Adamant",
+				"moves": [["Earthquake"], ["Knock Off"], ["Rapid Spin"], ["Toxic", "Ice Shard"]]
+			}]
+		},
+		"doublade": {
+			"flags": {},
+			"sets": [{
+				"species": "Doublade",
 				"gender": "",
-				"item": ["Rocky Helmet"],
-				"ability": ["Bulletproof"],
-				"evs": {"hp": 252, "atk": 0, "def": 84, "spa": 0, "spd": 0, "spe": 172},
-				"nature": "Impish",
-				"moves": [["Spikes"], ["Taunt"], ["Synthesis"], ["Wood Hammer", "Drain Punch"]]
+				"item": ["Eviolite"],
+				"ability": ["No Guard"],
+				"evs": {"hp": 252,"atk": 180,"def": 0,"spa": 0,"spd": 76,"spe": 0},
+				"nature": "Adamant",
+				"moves": [["Swords Dance"], ["Shadow Sneak"], ["Sacred Sword"], ["Shadow Claw", "Toxic", "Pursuit"]]
 			}]
 		},
-		"comfey": {
+		"drapion": {
 			"flags": {},
 			"sets": [{
-				"species": "Comfey",
+				"species": "Drapion",
 				"gender": "",
-				"item": ["Pixie Plate"],
-				"ability": ["Triage"],
-				"evs": {"hp": 96, "atk": 0, "def": 0, "spa": 252, "spd": 0, "spe": 160},
-				"nature": "Modest",
-				"moves": [["Draining Kiss"], ["Leech Seed"], ["Taunt"], ["Synthesis"]]
+				"item": ["Choice Band"],
+				"ability": ["Sniper"],
+				"evs": {"hp": 0,"atk": 252,"def": 4,"spa": 0,"spd": 0,"spe": 252},
+				"nature": "Jolly",
+				"moves": [["Earthquake"], ["Knock Off"], ["Poison Jab"], ["Pursuit"]]
+			}, {
+				"species": "Drapion",
+				"gender": "",
+				"item": ["Groundium Z", "Waterium Z"],
+				"ability": ["Sniper"],
+				"evs": {"hp": 0,"atk": 252,"def": 4,"spa": 0,"spd": 0,"spe": 252},
+				"nature": "Jolly",
+				"moves": [["Swords Dance"], ["Knock Off"], ["Poison Jab"], ["Earthquake", "Aqua Tail"]]
```

Comment:
Don't do this either. I had already specifically split this set into two; Groundium Z with Earthquake and Waterium Z with Aqua Tail.

Reply:
You seem to be redoing everything instead of adding to it. The Shaymin EVs were fine before too.